CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ Sharpbrake Changelog
 
 ### master
 
+### [v4.5.0][v4.5.0] (January 12, 2018)
+
+* Updated NLog dependencies
```

Comment:
Not sure why it would be interesting for library users (perhaps I'm wrong, though?)

Reply:
Probably, it's not worth to mention. Will remove...